bundles/org.openhab.binding.bluelink/src/main/resources/OH-INF/thing/vehicle.xml

```diff
@@ -0,0 +1,308 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluelink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="vehicle">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="account"/>
+		</supported-bridge-type-refs>
+
+		<label>Bluelink Vehicle</label>
+		<description>A Hyundai vehicle connected via Bluelink.</description>
+
+		<channel-groups>
+			<channel-group id="status" typeId="status"/>
+			<channel-group id="charging" typeId="charging"/>
+			<channel-group id="fuel" typeId="fuel"/>
+			<channel-group id="doors" typeId="doors"/>
+			<channel-group id="windows" typeId="windows"/>
+			<channel-group id="climate" typeId="climate"/>
+			<channel-group id="range" typeId="range"/>
+			<channel-group id="warnings" typeId="warnings"/>
+		</channel-groups>
+
+		<properties>
+			<property name="vin">Vehicle Identification Number</property>
+			<property name="model">Vehicle Model</property>
+			<property name="engineType">Engine Type</property>
+		</properties>
+
+		<representation-property>vin</representation-property>
+
+		<config-description>
+			<parameter name="vin" type="text" required="true">
+				<label>VIN</label>
+				<description>Vehicle Identification Number (VIN).</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" min="0" unit="min" required="false">
+				<label>Refresh Interval</label>
+				<description>Interval in minutes to fetch cached data from the Hyundai/Kia servers.</description>
+				<default>30</default>
```

Comment:
This conflicts with the readme. That states 60 minutes, either one needs to be changed.